Dark chocolate raspberry cheesecake

eytanadler opened the pull request:
Here's a new challenge to put our PR validation process to the test!

eytanadler committed:
Added chocolate raspberry cheesecake recipe

eytanadler committed:
Added cheesecake to dessert index

bernardopacini commented:
Woohoo! Looking forward to it. You'll guide us through all 3 days right?? Jenna has a spare room

eytanadler commented:
> Woohoo! Looking forward to it. You'll guide us through all 3 days right?? Jenna has a spare room

Sure! We might have to take some days off work though…

jennank commented on source/dessert/chocolateCheesecake.rst:
```diff
@@ -0,0 +1,113 @@
+=====================
+
+#. In the bowl of a food processor fitted with a metal blade, process the raspberries for 10 seconds.
+#. Heat 1 inch of water in the bottom half of a double boiler over medium high heat. Place 16 ounces semisweet chocolate, 1/3 cup coffee, and the pureed raspberries in the top half of the double boiler. Heat uncovered for 8 to 10 minutes, then remove from the heat and stir until smooth.
```
When is this chocolate mixture added to the cream cheese blend?

eytanadler replied:
Excellent catch! I missed a sentence

ntemplon reviewed (commented):
I rate this cheesecake 215mg/dL out of 10. LGTM.

bernardopacini commented:
> I rate this cheesecake 215mg/dL out of 10. LGTM.

Diabetics check: Passed

jennank commented on source/dessert/chocolateCheesecake.rst:
```diff
@@ -0,0 +1,113 @@
+Chocolate-Raspberry Cheesecake
+-------------------------------
+* 1/2 pint red raspberries
+* 16 ounces semisweet chocolate, broken into 1/2-ounce pieces
```
Is there a reason some are "broken into 1/2-oz pieces" while others are "chopped into 1/2-inch pieces"?

eytanadler replied:
🤷 I don't make the rules

jennank commented on source/dessert/chocolateCheesecake.rst:
```diff
@@ -0,0 +1,113 @@
+#. Combine together in a sifter 3/4 cup flour, 1/2 teaspoon baking soda, and 1/2 teaspoon salt. Sift the mixture onto parchment paper and set aside.
+#. In the bowl of a food processor fitted with a metal blade, combine 3 ounces semisweet chocolate, 2 ounces unsweetened chocolate, 1/2 cup granulated sugar, and 1/4 cup light brown sugar. Process for 20 seconds. Add 4 tablespoons butter and 1 egg and process for 10 seconds. Add 2 teaspoons vanilla extract and process for 20 seconds. Scrape down the sides of the bowl.
+#. Add the sifted flour mixture to the chocolate mixture in the food processor, then pulse for 20 seconds.
+#. Line two baking sheets with parchment paper and preheat the oven to 350 degrees. Portion 12 tablespoons batter, 2 inches apart, onto each of the two baking sheets. Bake in the preheated oven for 22 to 25 minutes. Allow the baked cookies to cool at room temperature for 20 minutes.
```
I found that it was more like 20 tablespoons, rather than 12. Are only 12 needed for the crust? or should all cookies be used no matter how many are made? 12 cookies only yielded around 2.5-3 cups crumbs

eytanadler replied:
I suspect they're big tablespoons and you just use all the cookies, but any beyond the 4 cups crumbs you get to eat :)

jennank commented on source/dessert/chocolateCheesecake.rst:
```diff
@@ -0,0 +1,113 @@
+#. Place 1 1/4 pounds softened cream cheese, 1 cup sugar, and 1 teaspoon salt in the bowl of an electric mixer fitted with a paddle. Beat on low speed for 1 minute, on medium for 1 minute, and on high for 1 minute. Scrape down the sides of the bowl, then beat on high for 2 more minutes. Scrape down the bowl, then beat for 2 additional minutes on high speed.
+#. Add the 4 eggs, one at a time, beating on high for 15 seconds and scraping down the bowl after each addition. Then beat for 1 more minute on high.
+#. Add 1 teaspoon vanilla extract and beat on high for 5 seconds.
+#. Remove the bowl from the mixer. Use a rubber spatula to finish mixing the batter until it is smooth and thoroughly combined.
```
I believe this is the point where the chocolate is added, but unsure.

eytanadler replied:
Nailed it

eytanadler commented:
> I rate this cheesecake 215mg/dL out of 10. LGTM.

Are we sure that's enough? Might need to do some unit testing on the components to figure out where to add some (syntactic?) sugar.

jennank commented on source/dessert/chocolateCheesecake.rst:
```diff
@@ -0,0 +1,113 @@
+---------------
+
+#. The cheesecake can now be released from the springform pan. Wrap a damp, hot towel around the sides of the pan (the towel should be large enough to completely wrap around and cover the sides of the pan) and hold around the pan for about 1 minutes. Carefully release and remove the springform pan.
+#. Optionally, garnish the top of the cheesecake with fresh, whole red raspberries.
```
This should be included as an optional ingredient in the ingredients list

eytanadler replied:
Yes chef

jennank commented on source/dessert/chocolateCheesecake.rst:
```diff
@@ -0,0 +1,113 @@
+#. Optionally, garnish the top of the cheesecake with fresh, whole red raspberries.
+#. Use a serrated knife to trim the crust so that it is level with the ganache (trim the crust away from the cake so that it does not fall into the ganache).
+#. Portion 3 to 4 tablespoons raspberry sauce onto each plate.
+#. Heat the blade of a serrated knife under hot running water before slicing into the cheesecake. Cute the cheesecake with the serrated knife, reheating after making each slice.
```
Typo: *"cute"* the cheesecake should be "cut"

eytanadler replied:
Good catch!

jennank commented on source/dessert/index.rst:
```diff
@@ -7,3 +7,4 @@ Dessert
    bananaBread
    strawberryCobbler
    strawberryPuddingPopsicle
+   chocolateCheesecake
```
Please follow the alphabetical convention.

eytanadler replied:
🫡

jennank reviewed (changes requested):
Overall things look good, but please reference the other recipes for proper formatting (header type, categories, etc.) A few minor other comments and clarifications, but a delicious recipe!

eytanadler committed:
Addressed Jenna's very good comments

eytanadler committed:
Merged main

eytanadler commented:
Gonna make this recipe again in a few weeks so I can give it a final pass before merging

eytanadler committed:
Improved cookie quantity specification

eytanadler committed:
Refinements after making it again

Updated time estimate and clarified instructions for cookie baking and cheesecake release.

eytanadler commented:
Just made it again and added some small tweaks! I think it’s officially ready

eytanadler committed:
Fix typos

bernardopacini commented:
Woohoo! Thanks @eytanadler! We'll give it a quick scan and merge it.

bernardopacini commented on source/dessert/chocolateCheesecake.rst:
```diff
@@ -0,0 +1,141 @@
+* 3 ounces semisweet chocolate, broken into 1/2-ounce pieces
+
+Equipment
+---------
```
Minor, but newline needed here for consistent formatting.

bernardopacini commented on source/dessert/chocolateCheesecake.rst:
```diff
@@ -0,0 +1,141 @@
+* 2 teaspoons pure vanilla extract
+
+Chocolate-Raspberry Cheesecake
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Minor, but newline needed here for consistent formatting.

eytanadler commented:
Done!
